### PR TITLE
[KBSWITCH] Add new Portuguese (pt-PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[KBSWITCH] Add new Portuguese (pt-PT) translation

### CODE REVIEW

File: base/applications/kbswitch/lang/pt-PT.rc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*
+ * PROJECT:     KBSWITCH for ReactOS
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ *              Copyright 2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```
Comment (binarymaster) [inline base/applications/kbswitch/lang/pt-PT.rc:5]:
```suggestion
 * PROJECT:     Keyboard Layout Switcher
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Portuguese (Portugal) resource file
 * TRANSLATOR:  Copyright 2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```

File: base/applications/kbswitch/lang/pt-PT.rc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+
+IDR_POPUP MENU
+BEGIN
+    POPUP "popup"
+    BEGIN
+        MENUITEM "&Preferências...", ID_PREFERENCES
+        MENUITEM SEPARATOR
+        MENUITEM "&Fechar a barra de idiomas", ID_EXIT
```
Comment (binarymaster) [inline base/applications/kbswitch/lang/pt-PT.rc:16]: Note that en-US text is just "Exit".

Comment (zecarlos1957) [reply base/applications/kbswitch/lang/pt-PT.rc:16]: This menu item complies with Win2003 Srv "&Fechar a barra de idiomas"

**Comment by zecarlos1957** [comment]
VirtualBox_WinSrv2003_08_06_2024_17_52_59

File: base/applications/kbswitch/lang/pt-PT.rc (hunk 2 above)
Comment (binarymaster) [reply base/applications/kbswitch/lang/pt-PT.rc:16]: Win2003 has no kbswitch.exe, it's ReactOS own component. See also https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=ID_EXIT

**Review by HBelusca — APPROVED** [review]

**Pull request merged**